libvast/vast/hash/uniquely_represented.hpp

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <array>
+#include <cstdint>
```

Comment:
```suggestion
#include <cstddef>
```

And then change back `std::size_t` to `size_t`, per the other discussion.